pkg/framework/plugins/nodeutilization/lownodeutilization.go

```diff
@@ -179,6 +181,16 @@ func (l *LowNodeUtilization) Balance(ctx context.Context, nodes []*v1.Node) *fra
 		)
 	}
 
+	if l.handle.ShouldExportMetrics() {
+		for key, rt := range thresholds {
+			for i, t := range rt {
+				for resource, value := range t {
+					metrics.LowNodeUtilizationThresholds.WithLabelValues(key, strconv.Itoa(i), resource.String()).Set(float64(value))
```

Comment:
There could be a new handler for setting metrics. E.g. `l.handle.Metric(metricsName).WithLabelValues` or similar that would be a no-op in case metrics are not enabled.